src/main/java/edu/byu/cs/autograder/git/GitHubAssignmentGrader.java

```diff
@@ -11,11 +11,20 @@
  * then this grader will give them full points.
  */
 public class GitHubAssignmentGrader {
+    private static final String RESUBMIT_PROMPT = " Push another commit to your repository and re-request grading to receive points for this assignment.";
```

Comment:
Good use of configurable string messages. I'm glad to see that they are giving clear instructions to the students to resubmit for credit.